src/Collection/utils.ts

```diff
@@ -0,0 +1,32 @@
+import { getCurrentNetworkURNProtocol } from '../ethereum/utils'
+import { CollectionAttributes, FullCollection } from './Collection.types'
+
+export function getDecentralandCollectionURN(
```

Comment:
Question: You think any of these methods might make sense on the `CollectionService` class? I'm not entirely sure as of right now, but if no, maybe we can think about clear boundaries of these file and the service.

My first intuition is that utils is fine in these cases and if we wanted to extract this, it'd make more sense to do a `CollectionBridge` of sorts with these methods (overkill)

Reply:
I'm not entirely sure where this should go, as I think that we would really need more methods like this to have a clear view on how to structure them. The main reason why I'm not sure about moving it into a service is because this method only builds a string that will be used for a collection, but it's not directly involved with collections at all.